Emby.Server.Implementations/ScheduledTasks/Tasks/CleanupCollectionAndPlaylistPathsTask.cs

```diff
@@ -116,7 +116,7 @@ private void CleanupLinkedChildren<T>(T folder, CancellationToken cancellationTo
         foreach (var linkedChild in folder.LinkedChildren)
         {
             var path = linkedChild.Path;
-            if (!File.Exists(path))
+            if (Path.HasExtension(path) ? !File.Exists(path) : !Directory.Exists(path))
```

Comment:
Files aren't required to have an extension

Reply:
correct, but all files that jellyfin can see should have one?